.devcontainer/.codespace.Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM mcr.microsoft.com/vscode/devcontainers/python:3.11
+
+# Install Docker CLI
+RUN apt-get update \
+  && apt-get install -y docker.io
```

Comment:
Does podman also work? If so can we use that instead by default here?

Reply:
But would the cli work with podman ? The cli too requires docker to be present right ?

Reply:
Oh! I am not very well aware of podman, I'll learn about it and get back to you.

Reply:
@Julian , seems like its not that easy to run podman in codespace, facing lots of permission errors

Reply:
@Julian , just wanted to update you that I am trying this but initially was facing some permission issues in the codespace while running podman. Thats resolved now but somehow bowtie is still not able to run the implementations, they are failing to start.

Reply:
Cool -- could also give up, it's not the biggest deal if we go with this as is certainly, I just asked in case it was an easy change!

Reply:
I'll try solving it once again today and if I am not able to do so then maybe we can merge this as is and if needed change it in the future

Reply:
@Julian , facing issue creating access to the podman socket inside the codespace. Ig we can merge it as it is for now.